core/kernel.rbs

```diff
@@ -1269,7 +1269,7 @@ module Kernel : BasicObject
   # pp returns argument(s).
   #
   def self?.pp: [T] (T arg0) -> T
-              | (untyped, untyped, *untyped) -> Array[untyped]
+              | (untyped, *untyped) -> Array[untyped]
```

Comment:
The second `untyped` was actually to skip the first overload if one argument is given.
Not sure if removing it causes another type error or not. (It won't on Steep.)

Reply:
Oh, I see, that's what it meant.

The original definition is as follows:

```rb
module Kernel
  def pp(*objs)
    objs.each {|obj|
      PP.pp(obj)
    }
    objs.size <= 1 ? objs.first : objs
  end
  module_function :pp
end
```

I became concerned because Steep reported the following:

```
The method parameter has different kind from the declaration `[T(353)(354)] (?untyped, ?untyped, *untyped) -> (T(353)(354) | ::Array[untyped] | nil)`(Ruby::DifferentMethodParameterKind)
```

Reply:
@ksss I mean the use-site. It depends on how the overloading is resolved. (Steep uses the first matching one, so passing one argument to `pp` resolves to the first overload anyways.)

Reply:
OK. I'll revert this commit later.